Tweetbook.Contracts/V1/Responses/PagedResponse.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Tweetbook.Contracts.V1.Responses
+{
+    public class PagedResponse<T>
+    {
+        public PagedResponse(){}
+
+        public PagedResponse(IEnumerable<T> data)
+        {
+            Data = data;
+        }
```

Comment:
Do you use this anywhere?

Reply:
Yes, in the PostsController i'm returning a PagedResponse.